ui/vite.config.ts

```diff
@@ -15,7 +15,7 @@ export default ({ mode }) => {
 
   return defineConfig({
     base: mode === 'mock' ? undefined : '/apps/grid/',
-    server: mode === 'mock' ? undefined : { https: true },
+    server: mode === 'mock' || mode === 'development' ? undefined : { https: true },
```

Comment:
Can we get rid of this? We do need https locally for testing push notifications in development.

Reply:
just reverted this.